conda/core/prefix_data.py

```diff
@@ -182,57 +181,39 @@ def is_writable(self):
             self.__is_writable = is_writable
         return self.__is_writable
 
+    # # REMOVE: ?
     def _has_python(self):
         return 'python' in self._prefix_records
 
-    def _load_site_packages(self):
-        python_record = next(
-            (prefix_rec for prefix_rec in itervalues(self.__prefix_records)
-             if prefix_rec.name == 'python'),
+    @property
+    def _python_pkg_record(self):
+        """Return the prefix record for the package python."""
+        return next(
+            (prefix_record for prefix_record in itervalues(self.__prefix_records)
+             if prefix_record.name == 'python'),
             None
         )
-        if not python_record:
-            return
-        prefix_graph = PrefixGraph(self.iter_records())
-        known_python_records = prefix_graph.all_descendants(python_record)
 
-        def norm_package_name(name):
-            return name.replace('.', '-').replace('_', '-').lower()
-
-        anchor_file_endings = ('.egg-info/PKG-INFO', '.dist-info/RECORD', '.egg-info')
-        conda_python_packages = dict(
-            ((af, prefix_rec)
-             for prefix_rec in known_python_records
-             for af in prefix_rec.files
-             if af.endswith(anchor_file_endings) and 'site-packages' in af)
-        )
+    def _load_site_packages(self):
```

Comment:
I still feel most of the code inside this func should go elsewhere to separate concerns.

The python specific handling should go elsewhere. But in the meantime this is a good trimming. I mean there should be no mention of anchor files on this file, cause that is irrelevant to prefix_data